config/core/test-configs.yaml

```diff
@@ -1797,6 +1797,11 @@ device_types:
     params:
       boot_timeout: '10'
 
+  qcs6490-rb3gen2:
```

Comment:
You can simply drop changes to this file, it's for the legacy system only, which is currently on life support. Better not to add anything new there at this point.

Reply:
OK. Dropped.